ov/profiles.go

```diff
@@ -449,7 +450,11 @@ func (c *OVClient) SubmitNewProfile(p ServerProfile) (err error) {
 		p.ManagementProcessor = mp
 	}
 
-	data, err := c.RestAPICall(rest.POST, uri, p)
+	if force {
+		querys["force"] = "all"
+	}
```

Comment:
This will give only two options either ignore all warning , or dont ignore.  There are other options like to ignore specific warning